common/landed_titles/01_landed_titles.txt

```diff
@@ -5926,11 +5926,11 @@ e_dragonblight = {
 					color2 = { 255 255 255 }
 				}
 			}
-			c_assmel = {
+			c_moonwatch = {
```

Comment:
I wouldn't advise to rename titles in code. You need to update all other entries of these titles.
If you completed renaming, it's fine, but it's extra work.

Reply:
I believe I have updated all related files for the titles. Did not find any errors in the log. Can you check if I covered all necessary files? The relevant changes were in c18db1b9b and 1a1670a58.

Reply:
Yes, German localization keeps it.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/162030916-a41964ee-66e5-477b-9580-d0ead9dd69d7.png)

</details>